manifest.json

```diff
@@ -1,8 +1,14 @@
 {
+  "version": "0.1.0",
   "domain": "bchydro",
   "name": "BC Hydro",
-  "documentation": "https://github.com/emcniece/hass-bchydro",
+  "documentation": "https://github.com/sascha432/hass-bchydro",
```

Comment:
This documentation link change is valid for the fork, but this pull request is merging it back into `emcniece/hass-bchydro` and as such the link probably shouldn't change.

Are you interested in maintainership? Maybe we could move this project to an organization instead of personal repositories.

Reply:
You should change that to your repo, wasn't sure if you are still active. I think it is a good idea that you look over any code changes, this is just the manifest I modified.